common/lib/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer.rb

```diff
@@ -54,9 +58,23 @@ def sanitize_links_and_mentions(text:, unsafe: false, format_html: true)
           sanitize_nwo_text(doc)
 
           mode = unsafe ? :UNSAFE : :DEFAULT
-          return doc.to_commonmark([mode] + COMMONMARKER_OPTIONS) unless format_html
 
-          doc.to_html(([mode] + COMMONMARKER_OPTIONS), COMMONMARKER_EXTENSIONS)
+          # If the text does not contain markdown, we need to use the HARDBREAKS option
+          commonmarker_render_options = if text.length > 1000
```

Comment:
I assume that the check for text length is because you're trying to work around the CodeQL issue... but since we're not really worried about the CodeQL failure here, this code is overly complex.

Can you simplify it back to what you had earlier that simply checks for markdown and if not uses the `HARDBREAKS` option?

I think that will simplify stuff quite a bit here...

Reply:
Hey @jeffwidman, thanks for the review. I've simplified the logic back to my initial commit.